dataactbroker/handlers/userHandler.py

```diff
@@ -390,7 +390,7 @@ def getPermssionList(self):
         queryResult = self.session.query(PermissionType).all()
         return queryResult
 
-    def createUserWithPassword(self,email,password,bcrypt,admin=False):
+    def createUserWithPassword(self,email,password,bcrypt,permission=1):
```

Comment:
Actually you would need to do 2^getPermissionId("agency_user") to do it this way, in general it's preferred to use grantPermission and removePermission to avoid this.